### ISSUE TITLE
ClusterManagmentError 

### ISSUE THREAD

**Issue by gourangakole** [cms-sw/genproductions#1640]
Hi @khurtado ..I get the following error today. is it a global issue or particular for me?
ClusterManagmentError : Some Jobs are in a Hold/... state. Error messages are below.Please try to investigate or contact the IT team.
        ClusterId 2029690 with HoldReason: Error from slot1@glidein_36105_147857476@g27n06.hep.wisc.edu: STARTER at 144.92.180.151 fai\
led to send file(s) to <192.170.227.118:9618>: error reading from /var/condor/.execute-links/slot01/dir_36099/glide_vIrxBp/execute/dir\
_1564/scale_pdf_dependence.dat: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <144.92.180.151:16288>
Please report this bug on https://bugs.launchpad.net/mg5amcnlo

Full log file:
/local-scratch/gkole/15Jan2015/genproductions/bin/MadGraph5_aMCatNLO/diphotonJets_5f_NLO_FXFX_Mgg80ToInf.log

**Comment by khurtado** [comment]
@gourangakole This doesn't seem like a system issue, but a madgraph one. You get the same error for the same subprocesses on multiple sites. Have you tried in other submission system like the LPC? My guess is that you will get the same error.

Basically, some madgraph subprocesses are missing `scale_pdf_dependence.dat` after the job has finished on the worker node (event though the exit code is 0), but the madgraph cluster manager is expecting condor to transfer that file to the submit node... hence it fails.

```
TransferOutput = "events.lhe.rwgt,reweight_xsec_events.output,scale_pdf_dependence.dat"
```

@kdlong Any idea on this one?

**Comment by gourangakole** [comment]
Thank you @khurtado for the insight. 
So Just to give more information to @kdlong as well. This is the continuation problem of https://github.com/cms-sw/genproductions/issues/1518 ...but what I just wanted to check if I switch off the 2nd process like below:

import model loop_sm-ckm_no_b_mass

generate p p > a a [QCD] @0
add process p p > a a j [QCD] @1
# add process p p > a a j j [QCD] @2

**Comment by kdlong** [comment]
@gourangakole Sorry I overlooked this. It certainly look like this problem again to me: 

https://hypernews.cern.ch/HyperNews/CMS/get/generators/3675.html

**Comment by kdlong** [comment]
@gourangakole was this solved?

**Comment by gourangakole** [comment]
Hello @kdlong ...I have modified the card accordingly
https://github.com/gourangakole/genproductions/blob/e863cdae8177ccc6abbea56660bc8320d8b114c7/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_run_card.dat
but I see there is no success ... its still running for more than 6 days ... 
Logifile can befound here:
https://gkole.web.cern.ch/gkole/public/MC_Contact/Madgraph5_aMCatNLO/CMSConnect/diphoton_19-02-2018/

**Comment by gourangakole** [comment]
Hi @kdlong ..it is solved. Can you please merged #1694 and then close this issue.
